Monika After Story/game/script-ch30.rpy

```diff
@@ -1986,9 +1988,23 @@ label ch30_reset:
     # undo stuff from above
     python:
         if store.mas_dockstat.retmoni_status is not None:
-            mas_resetCoffee()
             monika_chr.remove_acs(mas_acs_quetzalplushie)
 
+        #We don't want to set up any drink vars/evs if we're potentially returning home this sesh
+        if store.mas_globals.returned_home_this_sesh:
```

Comment:
previous coffee logic did this undo paradigm, but this is a chance to do this correctly.

If we don't want to run consumable checking logic if we're in returned home flow, we should move the check somewhere else. `ch30_post_exp_check` seems alright since we do other standard checks (reactions/Affection/Apologies) in there.

Anything that should be explicitly removed because of a dockstat return can stay here in the `retmoni_status is not None` block. Avoid extra branches that check basically the same thing.

Code that needs to run after checkConsumables could be moved to `ch30_post_exp_check` as well, or if it should always be checked, move it to checkConsumables.